packages/map-template/src/components/OpeningHours/OpeningHours.jsx

```diff
@@ -0,0 +1,141 @@
+import { memo, useCallback, useState, useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import PropTypes from 'prop-types';
+import './OpeningHours.scss';
+import { ReactComponent as ClockIcon } from '../../assets/clock-light.svg';
+import { ReactComponent as ChevronDownIcon } from '../../assets/chevron-down.svg';
+import { ReactComponent as ChevronUpIcon } from '../../assets/chevron-up.svg';
+
+OpeningHours.propTypes = {
+    openingHours: PropTypes.object,
+    isAmFormat: PropTypes.bool,
+    isMondayFirstDayOfTheWeek: PropTypes.bool
+};
+
+/**
+ * 
```

Comment:
We need a general description of this component.

And also JSDoc to all function declarations in the component.